chapters/06-smart-contracts-in-ralph.md

```diff
@@ -384,7 +384,7 @@ Each address type is followed by a specific content bytes format:
 - `P2PKH` - serialized public key hash
 - `P2MPKH` - serialized public key hashes and multisig threshold
 - `P2SH` - serialized script hash
-- `P2C` - serialized contract id
+- `P2C` - serialized contract ID
```

Comment:
I noticed that in some places the guide uses uppercase and in others lowercase for ID. I replaced the lowercase with uppercase for consistency. Let me know if you agree!